util/measures/2018/qcdr-measures-strata-names.json

```diff
@@ -0,0 +1,42 @@
+{
+  "AHSQC6":["overall","<=10cm",">10cm"],
```

Comment:
@NateTheGreatt @kalvinwang Are the strata names suppose to match the 2017 QCDR measure strata names when it's the same measure in both years?  I'm finding a few mismatches for example in 2017 for measure AHSQC6 (https://github.com/CMSgov/qpp-measures-data/blob/master/util/measures/2017/qcdr-measures-strata-names.json) the strata names were: `"AHSQC6": ["hernia", "overall", "hernia>10cm"]`

Should we use similar notation `"hernia>10cm"` and `"hernia<=10cm"`

Reply:
I tried to align the names with the strata names that were embedded inside of the `quality-strata.csv` file in this ticket: https://jira.cms.gov/browse/QPPA-2098

Reply:
Is there any benefit to keeping the strata names consistent (when possible) from year to year?  For example, do we know if data from the same strata name will be compared across years programatically?

Reply:
yes - if the strata itself doesn't change then we should keep them the same. we manually name these and I thought I had gone through and aligned them. are there other ones that aren't matched? let me know and I can re-review the document.

Reply:
In the case of AHSQC6, for 2018 we were told rate 1 was the overall, which is 2017 was named "hernia" so in this particular case the names changed for 2 of the 3 rates.

Reply:
@sarahnw There is also a hard-limit of 20 characters for the strata names set in the code, currently. The strata names for measureId `PP4` in the previously mentioned `quality-strata.csv` file breach this validation rule. @kalvinwang should this be increased?

Reply:
oh interesting I didn't know that. we can change the names! that measure was trickier to name... what about:
counselalcohol
counseltobacco
counselsubstance
counselactivity

Reply:
@NateTheGreatt hm not sure why that 20char limit exists. i think we can leave that for a future pr since the current names are fixed now. thanks for the heads up